### PR TITLE
[zh] Fix typo in TLS Secrets part

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/44251/files

### CODE REVIEW

File: content/zh-cn/docs/concepts/configuration/secret.md [hunk 1]
```diff
@@ -645,7 +645,7 @@ Kubernetes 所提供的内置类型 `kubernetes.io/ssh-auth` 用来存放 SSH 
 <!--
 The SSH authentication Secret type is provided only for convenience.
 You can create an `Opaque` type for credentials used for SSH authentication.
-However, using the defined and public Secret type (`kubernetes.io/ssh-auth`) helps other
+However, using the defined and public Secret type (`kubernetes.io/tls`) helps other
```
Comment (dipesh-rawat) [inline content/zh-cn/docs/concepts/configuration/secret.md:648]: This fix should also be extended to [Line 655](https://github.com/kubernetes/website/blob/bbd2d2c4106c2e892b3561b763f8e8a72d2ad875/content/zh-cn/docs/concepts/configuration/secret.md?plain=1#L655) as well.

```diff
- 不过，使用预定义的、公开的 Secret 类型（`kubernetes.io/ssh-auth`）
+ 不过，使用预定义的、公开的 Secret 类型（`kubernetes.io/tls`）
```

Comment (windsonsea) [inline content/zh-cn/docs/concepts/configuration/secret.md:648]: Line 655 in zh para shall be changed too.

Comment (Arhell) [reply content/zh-cn/docs/concepts/configuration/secret.md:648]: Done, thanks

**Review by windsonsea — COMMENTED** [review]
/approve

No more in-synced content after checking:

- [en history](https://github.com/kubernetes/website/commits/main/content/en/docs/concepts/configuration/secret.md)
- [zh history](https://github.com/kubernetes/website/commits/main/content/zh-cn/docs/concepts/configuration/secret.md)

**Pull request merged**